Assets/MultiBuild/Editor/SettingsWindow.cs

```diff
@@ -266,10 +283,10 @@ public class SettingsWindow : EditorWindow {
 
         void UpdateTargetsNotAdded() {
             _targetNamesNotAdded = new List<string>();
-            foreach (var target in Targets) {
-                if (!Settings.targets.Contains(target)) {
+            for (int i = 0, n = (int)Target.Max; i < n; ++i) {
+                Target target = Targets[i];
+                if (!Settings.targets.Contains(target))
```

Comment:
Ah sorry I was reviewing on iOS and missed that you'd added `Max` to the end of the enum, I thought it was the Linq `Max` which is the highest value, which would have missed the last one.

I still don't see why `foreach` wouldn't iterate correctly though. Will check out after merge.